pkg/settings/settings.go

```diff
@@ -0,0 +1,42 @@
+package settings
+
+import (
+	"errors"
+	"os"
+	"strconv"
+)
+
+// Global
+var Settings = _Settings{}
+
+// Settings
+//   Plan: Plan settings.
+type _Settings struct {
+	Plan Plan
+}
+
+// Load settings.
+func (r *_Settings) Load() error {
+	err := r.Plan.Load()
+	return err
+}
+
+// Get positive integer limit from the environment
+// using the specified variable name and default.
+func getEnvLimit(name string, def int) (int, error) {
```

Comment:
Nice touch defaulting, wonder if spelling out `default` here would be easier to read since `def` means define in some languages